phpstan.neon.dist

```diff
@@ -164,13 +164,6 @@ parameters:
                 - src/Core/Checkout/Order/Aggregate/OrderDelivery/OrderDeliveryDefinition.php
                 - src/Core/System/NumberRange/NumberRangeDefinition.php
 
-        - # NEXT-25276 - Needs to be fixed with the update ot PHPUnit 10
```

Comment:
should not be removed. as this is not related to the changes right? also PHPStan will show errors, if you are on current trunk :wink: